src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,69 @@
+package qna.domain;
+
+import qna.CannotDeleteException;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class Answers {
+    private final List<Answer> answers;
+
+    public static Answers of(List<Answer> answers) {
+        return new Answers(answers);
+    }
+
+    private Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    public Answers() {
+        answers = new ArrayList<>();
+    }
```

Comment:
```suggestion
    private Answers(List<Answer> answers) {
        this.answers = answers;
    }

    public Answers() {
        answers = new ArrayList<>();
    }

    public static Answers of(List<Answer> answers) {
        return new Answers(answers);
    }
```

생성자 후 정적 팩토리 메서드가 오는 것이 관례입니다 :) 

```
class A {
    상수(static final) 또는 클래스 변수
    인스턴스 변수
    생성자
    팩토리 메소드
    메소드
    기본 메소드 (equals, hashCode, toString)
}
```

Reply:
넵 작성 해주신 관례를 따라서 코딩 해보겠습니다~